File: codigos/1 - estrutura sequencial/1002 - Area do Circulo/Main.clj
(def PI 3.14159)

(defn calcular-area [raio]
    (format "%.4f" (* PI (Math/pow raio 2))))

(defn main []
    (let [raio (read)
          area (calcular-area raio)]
        (println (str "A=" area))))

(main)

File: codigos/1 - estrutura sequencial/1002 - Area do Circulo/Main.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static String getArea(double raio){
        double area = raio * raio * 3.14159;
        return String.format("%.4f",area);
    }
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner prompt = new Scanner(System.in);
        double raio = prompt.nextDouble();
        System.out.println("A=" + getArea(raio));
    }
}